cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter uses the Roles and Organization Management outlined in [sig-governance](https://github.com/spinnaker/governance/blob/master/governance.md).
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for improving the experience of those who contribute upstream to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues](https://github.com/spinnaker/spinnaker/issues), we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Projects & Services
+
+-  In accordance with the code of conduct, establish policies, standards, and procedures for the use, moderation, and management of all public platforms officially used by the project, including but not limited to:
+  - [GitHub Management](https://github.com/spinnaker)
+  - [Slack](https://join.spinnaker.io/)
+  - [Spinnaker](https://www.youtube.com/channel/UCcxQbw8kT1-FRhFhO2QCetg) YouTube channel
+  - Zoom
+  - Google Meet
+- Set up alerting automation for GitHub events, including new contributors, new issues, and new pull requests.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub Management](https://github.com/spinnaker) for more detail
```

Comment:
I'm interested in more detail, but I don't see how the organization homepage would be providing any more clarity / details?

Reply:
This is a good catch. We need to add more clarification and reference prior art from other projects as needed (which is how this oversight came about in the first place, lol).

Reply:
@prophen @ExitoLab this is an easy fix, please check the referenced doc in the K8S project (and consult other porjects if you want), then you can just define whatever you want to say here instead of referencing another link. You can delete this bullet entirely if you want but I think that executing some GitHub mgmt tasks as described might be in scope - your call.